test/e2e/logforwarding/fluent/fluent_secure_test.go

```diff
@@ -103,13 +98,18 @@ var _ = Describe("[ClusterLogForwarder]", func() {
 		clf.Spec.Pipelines = []loggingv1.PipelineSpec{{InputRefs: []string{loggingv1.InputNameApplication}}}
 		for _, s := range f.Receiver.Sources {
 			secret := &loggingv1.OutputSecretSpec{Name: s.Name}
+			var tls *loggingv1.OutputTLSSpec
 			if s.Name == "no-auth" {
 				secret = nil
+				tls = &loggingv1.OutputTLSSpec{
+					InsecureSkipVerify: true,
+				}
```

Comment:
The no-auth case doesn't need skipverify, there's nothing wrong with the cert. I think the the problem was that the absence of a Secret was disabling TLS, which was incorrect. 
You'll need an additional mutual-auth case where the server has a bad cert to validate the skipverify flag works.

Reply:
The code never disabled TLS, the `TLS(false)` option was just a bit misleading.

I think I had pushed the last change just when you were doing the review. It seems one comment is on the old version and one on the new state of the PR. Maybe this caused some confusion?